pages/_app.js

```diff
@@ -46,6 +57,7 @@ function MyApp({ Component, pageProps }) {
             onDeleteSection={handleDeleteSection}
             onAddSection={handleAddSection}
             onAddExerciseSet={handleAddExerciseSet}
+            onDeleteSet={deleteSet}
```

Comment:
you could name this "handleDeleteSet" because that's the "Konvention", when you pass something with "on"